config/i18n-tasks.yml

```diff
@@ -40,6 +40,11 @@ data:
   #     object_nl: "\n"
   #     array_nl: "\n"
 
+locales:
+  <% (ENV["ENFORCED_LOCALES"].presence&.split(",") || []).each do |path| -%>
+  <%= ENV["ENFORCED_LOCALES"] %>
```

Comment:
Yes definitely, good spot!

Based on your review I also noticed it was missing the YAML list indicator and I also decided to enclose the locales with single quotes to avoid errors with locales such as `no`.